### PR TITLE
Fix/Upgrade Deno dependencies

### PR DESCRIPTION
**Pull request by ethan605**
# Problem

The build step for the mock server of UI tests complains about dependencies hash mismatched: https://github.com/onfido/onfido-sdk-ui/runs/4218205011?check_suite_focus=true#step:9:515

# Solution

- Remove specific versions in `deps.ts`
- Rewrite `lock.json` & upgrade dependencies
- Migrate outdated dependencies

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [n/a] Has the CHANGELOG been updated?
- [n/a] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the TESTING_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [n/a] Have any new automated tests been implemented or the existing ones changed?
- [n/a] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Review by stefaniacardenas — APPROVED** [review]

**Review by kopijunkie — APPROVED** [review]

File: test/mock-server/jwt.ts [hunk 1]
```diff
@@ -29,4 +29,12 @@ const header: Header = {
   alg: 'HS256',
 }
 
-export const generateToken = () => create(header, payload, '')
+export const generateToken = async () => {
+  const key = await crypto.subtle.generateKey(
+    { name: 'HMAC', hash: 'SHA-256' },
+    true,
+    ['sign', 'verify']
+  )
```
Comment (stefaniacardenas) [inline test/mock-server/jwt.ts:37]: Why is this needed?

Comment (ethan605) [reply test/mock-server/jwt.ts:37]: It's an update from `djwt` (https://github.com/timonson/djwt). Before this, the encryption function took a `string` key. But from the new versions, it uses `CryptoKey` which need to be initialised this way
